src/components/FilterBar/FilterBar.stories.mdx

```diff
@@ -117,12 +117,22 @@ The passed in width will take precedence over the default behavior.
 Note: It's suggested that the global Apply and Clear buttons be used with `listbox` MultiSelect variants.
 MultiSelect `dialog` variants render an Apply and a Clear button within each MultiSelect component.
 
+### Standalone Examples
+
 <Canvas>
   <DSProvider>
     <story.FilterBarLayoutStory />
   </DSProvider>
 </Canvas>
 
+### Examples With Containers
+
+<Canvas>
+  <DSProvider>
+    <story.FilterBarLayoutContainerStory />
```

Comment:
I think it would be good to show how the background element was created. It seems to be a simple `Box` with some styles. This pattern is part of the UI in a page in the Research Catalog so it can be a reusable pattern with code that can/should be copied and pasted (or perhaps an actual wrapper ds component?).

Reply:
I was considering making the wrapper (and the background treatment) part of the component itself.  Maybe a `wrapper` prop with the following values: 

- `none`: this would match what is currently implemented
- `default`: this would match the examples that I created
- `fullWidth`: this would allow the wrapper to span the full window width and keep the content within 1280px (similar to the Breadcrumbs component)

`fullWidth` would really only be applicable for the `row` layout, so maybe that's not the best choice.  Perhaps we just implement an `isWrapped` prop and allow the consuming app to place the `FilterBar` component accordingly in the `Template` layout.

What do you think?

Reply:
Yeah, that sounds good although that might overcomplicate this component? Is there a need for other components to have a similar looking background pattern? If it's just the FilterBar, then a `isWrapped` prop is fine. I was thinking if other components need a similar background pattern, then it could be a new component.

Reply:
OK, I understand and I like the idea of handling the UI pattern with a DS component.  That said, I think it would make the most sense to extend the functionlaity of the existing `ComponentWrapper` component to do this.

For now, I will update the examples here to show the current code that I have used.  I will also take a closer look at `ComponentWrapper` and see what it woudl take to extenbd the existing functionality and then create the necessary tickets.